src/goes/test_goes_converter.py

```diff
@@ -30,6 +35,8 @@ def test_goes_converter(input_file_paths, latlon_file_path, output_file_path_rf,
     latlon_file_path = sys.argv[2]
     output_file_path_rf = sys.argv[3]
     output_file_path_bt = sys.argv[4]
-    test_goes_converter(input_file_paths, latlon_file_path, output_file_path_rf, output_file_path_bt)
+    resolution = sys.argv[5]
+    resolution = sys.argv[6]
```

Comment:
It appears that you have an extra assignment statement here and I can only assume that you need only one of these two - not sure which one.

Reply:
This has been resolved.